.github/workflows/ci.yml

```diff
@@ -12,10 +12,10 @@ jobs:
   node_test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.3.0
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
       - id: npm-cache-dir
         run: echo "dir=$(npm config get cache)" >> $GITHUB_OUTPUT
-      - uses: actions/cache@v3.3.1
+      - uses: actions/cache@69d9d449aced6a2ede0bc19182fadc3a0a42d2b0 # v3.2.6
```

Comment:
👍 Forgot about this newer SHA pinning _posture_

Reply:
Yeah, there's a CLI from security that will pin it for you based on the tag. Will send you a link.